lib/vladlev.rb

```diff
@@ -1,11 +1,12 @@
 module Vladlev
-  def self.relative_exists?(filename)
-    File.exists?(File.join(File.dirname(__FILE__), filename))
+  def self.relative_existing_path(filename)
+    file_path = File.join(File.dirname(__FILE__), filename)
+    File.exists?(file_path) && file_path
   end
 
-  JRUBY_NATIVE = RUBY_PLATFORM =~ /java/ && relative_exists?('levenshtein.jar')
+  JRUBY_NATIVE = RUBY_PLATFORM =~ /java/ && relative_existing_path('levenshtein.jar')
   C_EXT_NATIVE = !JRUBY_NATIVE &&
-    (relative_exists?('levenshtein.bundle') || relative_exists?('levenshtein.so'))
+    (relative_existing_path('levenshtein.so') || relative_existing_path('levenshtein.bundle'))
```

Comment:
I don't know how it's suppose to work. But you're not the only one referencing the `.bundle`.
https://github.com/dbalatero/levenshtein-ffi/blob/master/lib/levenshtein.rb#L9